src/webview/cluster/clusterViewLoader.ts

```diff
@@ -185,6 +190,22 @@ async function clusterEditorMessageListener (event: any ): Promise<any> {
     }
 }
 
+async function pollClipboard(signupStatus) {
+    while (panel) {
+        const previousContent = await vscode.env.clipboard.readText();
+        await new Promise(r => setTimeout(r, 500));
+        const currentContent = await vscode.env.clipboard.readText();
+        if (previousContent && previousContent !== currentContent) {
+            const oauthInfo = await sandboxAPI.getOauthServerInfo(signupStatus.apiEndpoint);
```

Comment:
Since sending and receiving the response from the auth server takes some time, the user could copy the token in this time. That means the change in content won't be detected.

I ran into this. As a result, I can't click the button unless I already have the token in my clipboard when I open the cluster view.